### PR TITLE
Docs 9647 consolidate worker size per v cores hn

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

File: modules/ROOT/pages/cloudhub-architecture.adoc [hunk 1]
```diff
@@ -20,33 +20,31 @@ Manage CloudHub with the Anypoint Runtime Manager console in Anypoint Platform.
...
-
 To understand CloudHub’s approach to security and availability, it’s important to understand the architecture behind CloudHub.  It includes two major components--Anypoint platform services, and the worker cloud. These two components and the Runtime Manager console through which you access them work together to run your integration applications.
 
 image::architecture-diagram.png[]
 
 [cols="10a,45a,45a"]
 |===
-|  |<<Integration Applications>> |These are applications that you create and deploy to CloudHub to perform integration logic for your business.
-|image:blue-1.png[] |<<Runtime Management Console>> |The Runtime Manager console is the face of CloudHub, allowing you to deploy and monitor integrations, and configure your account, among other things.
+|  |<<Integration Applications>> |The applications that you create and deploy to CloudHub to perform integration logic for your business
```
Comment (valkyrie69) [inline modules/ROOT/pages/cloudhub-architecture.adoc]: Not a big deal, but I would take "The" out of these descriptions to stay parallel and avoid The The The

File: modules/ROOT/pages/cloudhub-architecture.adoc [hunk 2]
```diff
@@ -92,19 +90,39 @@ You can scale workers vertically by selecting one of the available worker sizes:
...
+
+Workers with 0.1 vCores and 0.2 vCores:
+
+* Provide limited CPU and I/O for apps with smaller workloads
+* Can burst to higher CPU speeds for a short time
++
+This ability helps to improve application startup times and to process infrequent, large workloads.
+If you need consistent performance, use workers with more vCores.
+
+Workers with 1 or more vCores:
```
Comment (valkyrie69) [inline modules/ROOT/pages/cloudhub-architecture.adoc:102]:
```suggestion
Workers with one or more vCores:
```

File: modules/ROOT/pages/cloudhub-architecture.adoc [hunk 3]
```diff
@@ -92,19 +90,39 @@ You can scale workers vertically by selecting one of the available worker sizes:
...
+Only running applications count toward worker usage.
+
+If your applications require more vCores than are available, CloudHub allows you to create the application, but you cannot start it until the additional vCores become available.
+To increase vCore allocation, stop another application or contact your account manager to increase your subscription's vCore allocation.
 
 The metaspace limit for apps deployed to CloudHub is currently 256 MB, regardless of the worker size.
 The initial metaspace size is 128 MB; metaspace garbage collection begins after the metaspace reaches that threshold.
 
-You can also scale your applications horizontally by adding multiple workers and enabling persistent queues to distribute workload across workers.
+You can scale your applications horizontally by adding multiple workers and enabling persistent queues to distribute workload across workers.
```
Comment (valkyrie69) [inline modules/ROOT/pages/cloudhub-architecture.adoc]: workloads?

File: modules/ROOT/pages/managing-applications-on-cloudhub.adoc [hunk 4]
```diff
@@ -73,7 +73,7 @@ Domains for applications deployed to Europe automatically update to `+http://mya
 
 == Scale Your Application
 
-You can dynamically scale your applications by selecting options from the *Worker size* drop-down menu, and clicking the *Apply Changes* button. Your application uses the worker size and amount that you specify. For more information about deploying to multiple workers, see xref:deploying-to-cloudhub.adoc#worker-size-and-vcores[Worker Size and vCores].
+You can dynamically scale your applications by selecting options from the *Worker size* drop-down menu, and clicking the *Apply Changes* button. Your application uses the worker size and amount that you specify. For more information about deploying to multiple workers, see xref:cloudhub-architecture.adoc#cloudhub-workers[CloudHub Workers].
```
Comment (valkyrie69) [inline modules/ROOT/pages/managing-applications-on-cloudhub.adoc]: "...worker size and amount..." ? Worker size and number of workers?

**Review by valkyrie69 — DISMISSED** [review]

File: modules/ROOT/pages/cloudhub-architecture.adoc (hunk 1 above)
Comment (hannanelson) [reply modules/ROOT/pages/cloudhub-architecture.adoc]: agreed

File: modules/ROOT/pages/cloudhub-architecture.adoc (hunk 2 above)
Comment (hannanelson) [reply modules/ROOT/pages/cloudhub-architecture.adoc:102]: left as 1 because ... table

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**